modules/RTC/RTCUtils.js

```diff
@@ -824,6 +866,35 @@ class RTCUtils extends Listenable {
                         return this.audioTracks;
                     };
                 }
+
+            } else if (RTCBrowserType.isEdge()) {
+                this.peerconnection = edgeRTCPeerConnection;
+                this.getUserMedia
+                    = wrapGetUserMedia(
+                        navigator.mediaDevices.getUserMedia.bind(
+                            navigator.mediaDevices),
+                            true);
+                this.enumerateDevices
+                    = wrapEnumerateDevices(
+                        navigator.mediaDevices.enumerateDevices.bind(
+                            navigator.mediaDevices));
+                this.attachMediaStream
+                    = wrapAttachMediaStream((element, stream) => {
+                        defaultSetVideoSrc(element, stream);
+
+                        return element;
+                    });
+
+                // TODO: needed in Edge?
```

Comment:
Are we waiting just for this? I mean: I don't know yet whether it may be needed or not, but I'm pretty sure it does not matter within the specific purpose of this PR.